user-storage-simple/src/main/java/org/keycloak/quickstart/writeable/PropertyFileUserStorageProvider.java

```diff
@@ -284,16 +181,96 @@ public void disableCredentialType(RealmModel realm, UserModel user, String crede
     private static final Set<String> disableableTypes = new HashSet<>();
 
     static {
-        disableableTypes.add(CredentialModel.PASSWORD);
+        disableableTypes.add(PasswordCredentialModel.TYPE);
     }
-
+    
     @Override
-    public Set<String> getDisableableCredentialTypes(RealmModel realm, UserModel user) {
+    public void close() {
 
-        return disableableTypes;
     }
+    
     @Override
-    public void close() {
-
+	public Stream<UserModel> searchForUserStream(RealmModel realm, String search) {
+    	return searchForUserStream(realm, search, 0, Integer.MAX_VALUE);
```

Comment:
This method has default implementation, this seems as redundant:

https://github.com/keycloak/keycloak/blob/main/server-spi/src/main/java/org/keycloak/storage/user/UserQueryProvider.java#L185-L187